EventView-1/src/main/java/com/eventview/service/EventService.java

```diff
@@ -27,4 +27,7 @@
 
 	List<Integer> getAllEventDay();
 
+	List<String> getTodayFNameEventType();
+
+	List<String> getUpcomingFNameEventType();
```

Comment:
explain this

Reply:
i will remove everything and re-think this